Automattic/wp-calypso
# chore(packages): Update postcss packages to their latest versions

#### Changes proposed in this Pull Request

* Update `postcss` and related packages to their latest versions
* Drop the script `autoprefixer`, as I think it is not used (can't find any reference to it in Calypso docs or the FG)
* Change how `postCssConfig` and `postCssOptions` work for Sass loader in `@automattic/calypso-build`


#### Testing instructions

* Run `yarn start` and make sure it compiles and there are no CSS errors.
* Load the live branch in IE11 and verify login still has styles (i.e. make sure #50231 doesn't regress)
